api/v1/swagger.yaml

```diff
@@ -4221,6 +4226,11 @@ paths:
                 pattern: ^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$
                 description: >
                   The id of the subject's parent.
+              parentAbsolutePath:
+                type: string
+                maxLength: 254
```

Comment:
Sorry I missed this before, but the field length is 4096 (longish) specified in db model. Please confirm that this maxLength is what we want.